### PR TITLE
Make scopes editable

### PR DESCRIPTION
**Pull request by ranasiddharth**
Change approved status(if True) of app to False after scope editing

### CODE REVIEW

File: views/application.py [hunk 1]
```diff
@@ -71,6 +72,18 @@ def update(self, request, *args, **kwargs):
                 new_team_member_ids.append(new_member.get('id'))
         partial = kwargs.pop('partial', False)
         instance = self.get_object()
+        """
+        If PATCH request contains dataPoints, then the dataPoints of the instance in
+        the db and the dataPoints in the request are converted to a set and compared.
+        If comparison returns False, then app's approved status is set to False.
+        """
+        if "data_points" in request.data:
+            if (set(instance.data_points) == set(request.data.get('data_points'))):
```
Comment (kmrinal19) [inline views/application.py]: The entire `if else` can be replaced with following
```
if not (set(instance.data_points) == set(request.data.get('data_points'))):
    instance.is_approved = False
```

File: views/application.py [hunk 2]
```diff
@@ -71,6 +72,18 @@ def update(self, request, *args, **kwargs):
                 new_team_member_ids.append(new_member.get('id'))
         partial = kwargs.pop('partial', False)
         instance = self.get_object()
+        """
+        If PATCH request contains dataPoints, then the dataPoints of the instance in
+        the db and the dataPoints in the request are converted to a set and compared.
+        If comparison returns False, then app's approved status is set to False.
+        """
+        if "data_points" in request.data:
```
Comment (kmrinal19) [inline views/application.py]: I think that this `if` statement can be clubbed with the `if` statement below

**Review by kmrinal19 — CHANGES_REQUESTED** [review]

File: views/application.py (hunk 1 above)
Comment (ranasiddharth) [reply views/application.py]: Changes made

File: views/application.py (hunk 2 above)
Comment (ranasiddharth) [reply views/application.py]: Changes made

Comment (ranasiddharth) [reply views/application.py]: Comments changed from doc string to block in final commit

**Review by kmrinal19 — APPROVED** [review]

**Review by rish9101 — APPROVED** [review]

**Pull request merged**